<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLineEdit" name="tell_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="receiver_sel_comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(152, 152, 152);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send_pbtn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>550</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(30, 186, 180);</string>
   </property>
   <property name="text">
    <string>보내기</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="comm_textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>481</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>sendcategoribtn_clicked()</slot>
 </slots>
</ui>
